src/components/JoinUs/OpenSource.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import remcalc from 'remcalc'
+import breakpoint from 'styled-components-breakpoint'
+import styled from 'styled-components'
+import getYoutubeId from 'get-youtube-id'
+import { Padding, Margin } from 'styled-components-spacing'
+import { CompensatedRow, CompensatedCol } from '../grid'
+import Image from '../Common/Image'
+import { Paragraph } from '../Typography'
+import playIcon from '../../images/button-play-default.svg'
+
+import { Section, Separator, TitleAndBody, TitleAndList } from './elements'
+
+const MAX_VIDEOS = 3
+
+const getThumbnail = videoId =>
+  `https://img.youtube.com/vi/${videoId}/maxresdefault.jpg`
+
+const TalkLink = styled.a`
+  border: 1px solid #eeefee;
+  display: block;
+  flex: 1;
+`
+const TalkTitle = styled.div`
+  display: flex;
+`
+const PlayIcon = styled.img`
+  min-height: ${remcalc(36)};
+  max-width: ${remcalc(36)};
+`
+const TruncatedParagraph = styled(Paragraph)`
+  max-height: 3rem;
+  overflow: hidden;
+
+  @supports (-webkit-line-clamp: 2) {
+    max-height: none;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+`
+
+const TalkColumn = styled(CompensatedCol)`
+  ${breakpoint('largePhone')`
+    &:last-child {
+      display: none;
+    }
+  `}
+
+  ${breakpoint('tablet')`
+    &:last-child {
+      display: flex;
+    }
+  `}
+`
+
+const Talks = ({ data }) => (
+  <CompensatedRow>
+    {data.slice(0, MAX_VIDEOS).map(({ title, link }, idx) => (
+      <TalkColumn width={[1, 1, 1, 6 / 12, 6 / 12, 4 / 12]} key={idx}>
+        <Padding top={2}>
+          <TalkLink px={2} py={2} href={link} target="_blank">
+            <Padding horizontal={2} vertical={2}>
```

Comment:
According to Zeplin, the CompactVideoLink component should have this padding around it:
![image](https://user-images.githubusercontent.com/15656538/51534764-92ba4f80-1e3e-11e9-8ac8-41ea09d0cca5.png)

Where does `24px` (i.e. `horizontal={2} vertical={2}`) come from?

Reply:
Hmm.... I was looking at `CompactVideoLink` because that's the trello ticket you picked up. But, looking at the page-level designs in Zeplin, it looks like we're actually using `StandAloneVideoLink` on this page?

In which case, these are the appropriate paddings:
![image](https://user-images.githubusercontent.com/15656538/51534925-09574d00-1e3f-11e9-975d-7a05cdf37d0e.png)

So we would have something like:
```
const StandAloneVideoLink = props => (
  <Wrapper> // i.e. layout with compensated row, columns, etc.
    <Padding
      vertical={1.5} // i.e. 18px
      left={14} // since 14px isn't a value in our spacing sizes
      right={2} // back to spacing values
    >
      <Padding vertical={57}>
        <CompactVideoLink />
      </Padding>
    </Padding>
  </Wrapper>
)
```

Or you could amalgamate the 2 `Padding` components that are sitting next to each other?